cmd/commands/commands.go

```diff
@@ -1157,22 +1167,32 @@ func executeChannelClose(ctxc context.Context, client lnrpc.LightningClient,
 
 		switch update := resp.Update.(type) {
 		case *lnrpc.CloseStatusUpdate_CloseInstant:
-			if req.NoWait {
-				return nil
-			}
+			fmt.Println("Channel close successfully initiated " +
+				"(potentially waiting for HTLCs to be " +
+				"resolved), waiting for close tx to be " +
```

Comment:
nit: can we determine if it's waiting for HTLCs to be resolved and print here?
Maybe pass this information through a new field of `InstantUpdate` proto message?

Reply:
ok added a new field, number of pending htlcs.